packages/core/security/core-security-common/kibana.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "shared-common",
+  "id": "@kbn/core-security-common",
+  "owner": "@elastic/kibana-core"
```

Comment:
Kibana security owns `@kbn/security-plugin-types-common`. Should we add them as co-owners to catch unintended changes to the types?

Reply:
Good remark, Any objections @elastic/kibana-security?